src/core/tsi/ssl_transport_security_utils.h

```diff
@@ -142,6 +144,23 @@ tsi_result SslProtectorUnprotect(const unsigned char* protected_frames_bytes,
                                  unsigned char* unprotected_bytes,
                                  size_t* unprotected_bytes_size);
 
+// Verifies that `crl` was signed by `issuer.
+// return: true if valid, false otherwise.
+bool VerifyCrlSignature(X509_CRL* crl, X509* issuer);
+
+// Verifies the CRL issuer and certificate issuer name match.
+// return: true if equal, false if not.
+bool VerifyCrlCertIssuerNamesMatch(X509_CRL* crl, X509* cert);
+
+// Verifies the certificate in question has the cRLSign bit present.
+// OpenSSL >= 1.1.1 return: true if cRLSign bit is present, false otherwise.
+// OpenSSL == 1.0.2 return: true always
```

Comment:
you should fix this comment when you fix this ;)

Reply:
Fixed

Reply:
if the test works with 1.0.2 you're good to go :)